app/assets/scripts/components/map.js

```diff
@@ -272,23 +275,12 @@ const Map = React.createClass({
     const isQuantile = scale.hasOwnProperty('invertExtent');
     let iterable = (isQuantile ? scale.range() : scale.domain()).sort();
 
-    const nameLookup = {'very low': 5, 'low': 4, 'medium': 3, 'high': 2, 'very high': 1};
-    const idLookup = {5: 'very low', 4: 'low', 3: 'medium', 2: 'high', 1: 'very high'};
-    let convertId = true;
+    let convertId = false;
     let category = get(this.props, 'overlay.category');
-    if (category) {
-      category = category.toLowerCase();
-      if (category === 'categorical') {
-        // check whether array contains all numbers, sort if so.
-        if (!iterable.some(isNaN)) {
-          iterable = iterable.sort();
-          convertId = false;
-        // if array contains any words and they are in the lookup table, convert to numbers and sort.
-        // if they aren't in the lookup table, leave them alone.
-        } else {
-          iterable = iterable.map((cat) => nameLookup[cat] || cat).sort();
```

Comment:
IMO the issue with this step is that it's normalizing and not validating. A good validation step would give you some indication whether or not the input matches the given circumstances as to apply the conversion mapping.